rest/system-config-get.rst

```diff
@@ -3,109 +3,159 @@ GET /rest/system/config
-            ],
-            "globalAnnounceServers": [
-              "default"
-            ],
-            "globalAnnounceEnabled": true,
-            "localAnnounceEnabled": true,
-            "localAnnouncePort": 21027,
-            "localAnnounceMCAddr": "[ff12::8384]:21027",
-            "maxSendKbps": 0,
-            "maxRecvKbps": 0,
-            "reconnectionIntervalS": 60,
-            "relaysEnabled": true,
-            "relayReconnectIntervalM": 10,
-            "startBrowser": false,
-            "natEnabled": true,
-            "natLeaseMinutes": 60,
-            "natRenewalMinutes": 30,
-            "natTimeoutSeconds": 10,
-            "urAccepted": -1,
-            "urUniqueId": "",
-            "urURL": "https://data.syncthing.net/newdata",
-            "urPostInsecurely": false,
-            "urInitialDelayS": 1800,
-            "restartOnWakeup": true,
-            "autoUpgradeIntervalH": 12,
-            "keepTemporariesH": 24,
-            "cacheIgnoredFiles": false,
-            "progressUpdateIntervalS": 5,
-            "limitBandwidthInLan": false,
-            "minHomeDiskFreePct": 1,
-            "releasesURL": "https://upgrades.syncthing.net/meta.json",
-            "alwaysLocalNets": [],
-            "overwriteRemoteDeviceNamesOnConnect": false,
-            "tempIndexMinBlocks": 10
-          },
-          "ignoredDevices": [],
-          "ignoredFolders": []
-        }
+      "version": 30,
+      "folders": [
+	{
+	  "id": "GXWxf-3zgnU",
+	  "label": "MyFolder",
+	  "filesystemType": "basic",
+	  "path": "...",
+	  "type": "sendreceive",
+	  "devices": [
+	    {
+	      "deviceID": "...",
+	      "introducedBy": ""
+	    }
+	  ],
+	  "rescanIntervalS": 60,
+	  "fsWatcherEnabled": false,
+	  "fsWatcherDelayS": 10,
+	  "ignorePerms": false,
+	  "autoNormalize": true,
+	  "minDiskFree": {
+	    "value": 1,
+	    "unit": "%"
+	  },
+	  "versioning": {
+	    "type": "simple",
+	    "params": {
+	      "keep": "5"
+	    }
+	  },
+	  "copiers": 0,
+	  "pullerMaxPendingKiB": 0,
+	  "hashers": 0,
+	  "order": "random",
+	  "ignoreDelete": false,
+	  "scanProgressIntervalS": 0,
+	  "pullerPauseS": 0,
+	  "maxConflicts": 10,
+	  "disableSparseFiles": false,
+	  "disableTempIndexes": false,
+	  "paused": false,
+	  "weakHashThresholdPct": 25,
+	  "markerName": ".stfolder",
+	  "copyOwnershipFromParent": false,
+	  "modTimeWindowS": 0
+	}
+      ],
+      "devices": [
+	{
+	  "deviceID": "...",
+	  "name": "Laptop",
+	  "addresses": [
+	    "dynamic",
+	    "tcp://192.168.1.2:22000"
+	  ],
+	  "compression": "metadata",
+	  "certName": "",
+	  "introducer": false,
+	  "skipIntroductionRemovals": false,
+	  "introducedBy": "",
+	  "paused": false,
+	  "allowedNetworks": [],
+	  "autoAcceptFolders": false,
+	  "maxSendKbps": 0,
+	  "maxRecvKbps": 0,
+	  "ignoredFolders": [],
+	  "pendingFolders": [
+	    "time": "2019-06-05T10:21:22+02:00",
+	    "id": "cpkn4-57ysy",
+	    "label": "SomeonesFolder"
+	  ],
```

Comment:
This is broken, might be why the syntax highlight doesn't kick in

Reply:
Quite right, that was a manual edit. Will fix ASAP.